### PR TITLE
Gutenboarding: 2 column design selector on > mobile viewport.

### LINKED ISSUE #42397: Gutenboarding: two column design picker grid on tablet missing
At Gutenboarding (`/new`), design picker goes straight to one column from 3 columns on tablets:

image

image

I expected to have 2 columns before narrower mobile version where we have 1 column:

image

It's also little problematic for mobile landscape:

image

@dubielzyk thoughts?

### PR DESCRIPTION
**Pull request by yansern**
#### Changes proposed in this Pull Request

* 2 column design selector on > mobile viewport.

#### Testing instructions

* Resize browser until design selector becomes 2 column.

#### Screenshot
image

Fixes #42397

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss [hunk 1]
```diff
@@ -74,6 +74,10 @@
 			row-gap: 48px;
 			margin: 0 0 30px;
 
+			@include break-medium {
+				grid-template-columns: 1fr 1fr;
```
Comment (alshakero) [inline client/landing/gutenboarding/onboarding-block/design-selector/style.scss:78]: I think the 64px gap is too big for mobiles. 

```suggestion
				grid-template-columns: 1fr 1fr;
				column-gap: 32px;
```

**Comment by simison** [comment]
Better!

Let's have it go to two columns in landscape mobiles, too:

image

Now I feel like it's going to 1 column little too early:

image

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss (hunk 1 above)
Comment (simison) [reply client/landing/gutenboarding/onboarding-block/design-selector/style.scss:78]: @dubielzyk you can maybe comment to that

Comment (dubielzyk) [reply client/landing/gutenboarding/onboarding-block/design-selector/style.scss:78]: Indeed too big. We should drop it down to 32px fairly quickly (tablet) and then 20px on mobile.

**Comment by yansern** [comment]
All mobile portraits will have 1 column. The rest will start with 2 columns.

File: client/landing/gutenboarding/onboarding-block/design-selector/style.scss (hunk 1 above)
Comment (simison) [reply client/landing/gutenboarding/onboarding-block/design-selector/style.scss:78]: @yansern Let's have the column gap go:

- 20px
- 32px tablet
- 64px desktop

**Pull request merged**